modules/agent-configuration-parameters.adoc

```diff
@@ -127,4 +129,12 @@ For more information, see "Root device hints" in the "Setting up the environment
 |The host network definition.
 The configuration must match the Host Network Management API defined in the link:https://nmstate.io/[nmstate documentation].
 |A dictionary of host network configuration objects.
-|====
\ No newline at end of file
+
+|minimalISO:
+|Defines whether the Agent-based Installer generates a full ISO or a minimal ISO image. When set to `True`, the Agent-based Installer generates an ISO without a rootfs image file, and instead contains details about where to pull the rootfs file from.
+
+When generating a minimal ISO, if a rootfs URL is not specified by the `bootArtifactsBaseURL` parameter, then the Agent-based Installer embeds a default URL that is accessible in environments with an internet connection.
```

Comment:
If this is a suggestion to switch to active voice, i think i'll try a rewording that's more like "if you do not specify a rootfs URL through the `bootArtifactsBaseURL` parameter...". I feel this wording would focus more on "taking action X, which you do with this parameter" rather than "using this parameter to take action X," which feels to me like it might distract users with the bootartifacts parameter and what else they could do with it.

Not sure if that explanation is very coherent  😅

Reply:
The aim of my revision was to avoid a dangling modifier. I think your suggested rewording is good!
You can read more about dangling modifiers in the "Grammar" section of the "Global audiences" topic in IBM Style: https://www.ibm.com/docs/en/ibm-style?topic=medium-global-audiences#grammar

Reply:
Makes sense, thanks for citing ISG stuff throughout this review, I find it very helpful!